cli/purge.cls.php

```diff
@@ -157,6 +157,7 @@ public function url($args)
 	{
 		$data = array(
 			Router::ACTION => Core::ACTION_QS_PURGE,
+			Router::VALIDATE_PURGE => preg_replace('/.{1,3}(.?)/', '$1', NONCE_KEY),
```

Comment:
What did this reg do? Why using this way? Could you please give some comments for code readability?